docs/Linuxbrew.md

```diff
@@ -0,0 +1,99 @@
+![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
+
+# Linuxbrew
+
+The Homebrew package manager may be used on Linux and Windows 10+, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about), as well as macOS. Homebrew is referred to as Linuxbrew when running on Linux or Windows. It can be installed in your home directory and does not require root access. Recent versions of *glibc* and *gcc* may be installed in your home directory on an older distribution of Linux. A Linuxbrew installation does not use any libraries provided by your host system, except *glibc* and *gcc* if they are sufficiently current.
+
+[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology).
+
+# Install
```

Comment:
I suggest using `## Install`: there should be only one header (`# Linuxbrew`) IMHO.

Reply:
In my opinion, `<title>` is for the title of the document, and `<h1>` (aka `#`) is for the first level of section headers.

Reply:
I agree with @maxim-belkin. This will look weird on docs.brew.sh otherwise.

Reply:
Alrighty. I'll give in… this time. =p